public/Lab-Files/Lesson02/Assignment/resume.html

```diff
@@ -0,0 +1,92 @@
+
+
+<p> <h4 style="color:CornflowerBlue;"><em>Aetna Healthcare Insurance<em> <br/> <a href="https://www.aetna.com/">Aenta</a> <br/>August 2014 - June 2015 <br/>
+(Healthcare Concier/ Affordable Care Act)<br/> Assisting members with benefit information about policy.
+Reviewing members benefits, claims, and billing information.
+Learning and researching of new members under Obama Care, 
+while keeping company within compliance of application time allowed. 
+Strong organization skills, working independently and maintaining time management.
+Going above and beyond to insure all needed info was received for our memebers and providers to insure health insurance was not loss during highly sensitive time.
+Excel, Microsoft, outlook express. 
+
+
+
+
+</p>
+
+<p><h4 style="color:CornflowerBlue;"> <em>Medtronic Diabetes </h4> <a href="http://www.medtronic.com/us-en/index.html">Medtronic</a>	<br/>November 2011- March 2013 <br/>
+Technical Support Representative(24hr Help Line)</em><br/>
+Assisted patients with technical support with their insulin pumps.
+Addressed safty of patient blood sugar, programming pump with basal rates and bolus settings.
+Assisted with trouble shooting methods of flow charts and concluding why pump was not delivering insulin. Strong attention to detail while assisting patient.
+Trained all aspects of Diabetes mellitus and explained to patients how to use the pump’s features to control diabetes.
+Met all call quality metrics and used Microsoft Excel, Word, Access. </p>
+
+<p> <h4 style="color:CornflowerBlue;"><em>Wells Fargo Company  </h4> <br/>  <a href="https://www.wellsfargo.com/"> Wells Fargo </a> <br/>                 	
+November 2010 - November   2011<br/>
+Loan Counselor </em><br/>
+Qualified buyers in their homes by offering financial counseling during temporary periods of financial difficulty. 
+Systems used: CPI (Computer Power, Inc,), Fidelity P309, Web Query, ER (Early Resolution), IAM, Soft Phone, Word, and Excel.
+Cured delinquencies in the one and two stages, primarily through various Loss Mitigation techniques after fully understanding the borrower's 
+reason or reasons for delinquency and their current future financial positions.
+
+<p> <h4 style="color:CornflowerBlue;"><em>Wells Fargo Company </h4><br/>	<a href="https://www.wellsfargo.com/"> Wells Fargo </a> 
+<br/>May 2008 August 2009 <br/>Customer Service Representative</em><br/>
+ssisted in personal banking (personal checking, savings, money market, cds, mortgage, loans, on-line banking and fraud)
+Provided valuable information on how products/services will meet customer needs.
+Responsible for building rapport and recognizing customers expressed and unexpressed needs.
+</p>
+
+
+
+<p> <h1 style="color:Blue"></h1>EDUCATION:<br/>
+South Side High School 
+High School Diploma</p>
+<a href="#top">Go to top of page</a>
+
+        
+        
+    
+        
+    
+    
+    
+    
+    
+    </body>
+    </html>
```

Comment:
Make sure you validate all of your assignments continuously. By doing so, you will avoid most of the mistakes you have made.
Also work on organizing, indenting, and spacing your code a bit better. You do not need so many blank lines, like from line 81-90. 
Indenting nested elements will help you keep better track of your tags, and it will also make it easier to read  your code.
Good job overall!